src/main/java/com/users/specification/UserSpecification.java

```diff
@@ -53,6 +53,30 @@ public Predicate toPredicate(Root<User> root, CriteriaQuery<?> query, CriteriaBu
             }
         }
 
+        if (criteria.getSecondaryEmail() != null) {
+            if (criteria.getSecondaryEmail().getEquals() != null) {
+                predicates.add(criteriaBuilder.equal(root.get("secondaryEmails"), criteria.getSecondaryEmail().getEquals()));
+            }
+            if (criteria.getSecondaryEmail().getContains() != null) {
```

Comment:
astea seamana cu specificatia de tip easy, folosesti tot like (numai acolo ai vrut tu sa iti arati talentu sa faci cu convert de clasa)

Reply:
pai am si like si equal si in, vrei sa las doar like, nu? oricum probabil ala o sa fie cel folosit